vllm/v1/core/kv_cache_utils.py

```diff
@@ -541,48 +571,113 @@ def _get_kv_cache_config_uniform_type(vllm_config: VllmConfig,
                 max_model_len_str, max_concurrency)
 
     per_layer_size = page_size * num_blocks
+    # All layers can be represented by the same virtual layer.
+    virtual_layer_map = [[layer_name for layer_name in kv_cache_spec]]
 
     kv_cache_config = KVCacheConfig(
         num_blocks=num_blocks,
         tensors={
             layer_name: KVCacheTensor(size=per_layer_size)
             for layer_name in kv_cache_spec
         },
-        groups=[[layer_name for layer_name in kv_cache_spec]],
-        kv_cache_spec=kv_cache_spec)
+        virtual_layers=create_virtual_layer(kv_cache_spec, virtual_layer_map),
+    )
     return kv_cache_config
 
 
-def get_kv_cache_configs(vllm_config: VllmConfig,
-                         kv_cache_specs: List[KVCacheSpec],
-                         available_memory: int) -> List[KVCacheConfig]:
+def make_uniform_kv_cache_type(kv_cache_spec: Dict[str, KVCacheSpec]):
```

Comment:
Maybe a better name: `treat_sliding_window_as_full_attention`

Reply:
Should be `treat_sliding_window_as_full_attention_for_hybrid_model` but it is too long. Does `unify_hybrid_kv_cache_specs` look better?